log/log.go

```diff
@@ -6,45 +6,133 @@
 package log
 
 import (
+	"flag"
 	"fmt"
-	golog "log"
+	"log"
+	"log/syslog"
 	"os"
 )
 
 // The following constants represent logging levels in increasing levels of seriousness.
 const (
+	// LevelDebug is the log level for Debug statements.
 	LevelDebug = iota
+	// LevelInfo is the log level for Info statements.
 	LevelInfo
+	// LevelWarning is the log level for Warning statements.
 	LevelWarning
+	// LevelError is the log level for Error statements.
 	LevelError
+	// LevelCritical is the log level for Critical statements.
 	LevelCritical
+	// LevelFatal is the log level for Fatal statements.
 	LevelFatal
 )
 
 var levelPrefix = [...]string{
-	LevelDebug:    "[DEBUG] ",
-	LevelInfo:     "[INFO] ",
-	LevelWarning:  "[WARNING] ",
-	LevelError:    "[ERROR] ",
-	LevelCritical: "[CRITICAL] ",
-	LevelFatal:    "[FATAL] ",
+	LevelDebug:    "DEBUG",
+	LevelInfo:     "INFO",
+	LevelWarning:  "WARNING",
+	LevelError:    "ERROR",
+	LevelCritical: "CRITICAL",
+	LevelFatal:    "FATAL",
 }
 
-// Level stores the current logging level.
-var Level = LevelDebug
+var (
+	// Level stores the current logging level.
+	Level = LevelInfo
+	// sysLogger is a syslog Writer connection.
+	sysLogger *syslog.Writer
+	// UseSyslog controls if syslog is used instead of stdout/err.
+	UseSyslog bool
```

Comment:
Yes, so that a user of the package can set them manually without relying on flag parsing. I would consider removing them entirely and instead only exporting `SetSyslog` (or a global `SysLogger *syslog.Writer` var), expecting the user to user `syslog.Dial` or `syslog.New` directly when enabling syslog.